packages/legend-query-builder/src/components/shared/LambdaEditor.tsx

```diff
@@ -189,9 +191,14 @@ const LambdaEditor_Inner = observer(
             : CODE_EDITOR_THEME.DEFAULT_DARK,
           ...lambdaEditorOptions,
         });
+        if (onEditorBlur) {
+          _editor.onDidBlurEditorText(() => {
```

Comment:
I thought we would do this outside of `useEffect()` so that we can call `cancel()` on `transformStringToLambda`?